src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
 
  public void myMethod() {
   if (CoNstANT.equals("abc")) {
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
**Severity**: INFO
**File**: src/main/java/se/bjurr/violations/lib/example/OtherClass.java L10

Must have at least one statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<479458641>*